### PR TITLE
Update BTS mainnet and testnet Nodes

### PR DESCRIPTION
**Pull request by iamredbar**

### CODE REVIEW

**Review by abitmore — APPROVED** [review]

File: src/config/config.js [hunk 1]
```diff
@@ -43,11 +43,16 @@ let _blockchains = {
         chainId: '39f5e2ede1f8bc1a3a54a7914414e3779e33193f1f5693510e73cb7a87617447',
         nodeList: [
             {
-                url: "wss://testnet.nodes.bitshares.ws",
```
Comment (sschiessl-bcp) [inline src/config/config.js:46]: Could you please add https://eu.nodes.testnet.bitshares.ws/

File: src/config/config.js [hunk 2]
```diff
@@ -5,34 +5,34 @@ let _blockchains = {
         chainId: '4018d7844c78f6a6c41c6a552b898022310fc5dec06da467ee7905a8dad512c8',
         nodeList: [
             {
-                url: "wss://eu.nodes.bitshares.ws"
```
Comment (sschiessl-bcp) [inline src/config/config.js:8]: This one is actually working :)

**Review by sschiessl-bcp — CHANGES_REQUESTED** [review]

File: src/config/config.js (hunk 1 above)
Comment (iamredbar) [reply src/config/config.js:46]: https or wss?

File: src/config/config.js (hunk 2 above)
Comment (iamredbar) [reply src/config/config.js:8]: I will re-add this one.

**Comment by iamredbar** [comment]
Updated with @sschiessl-bcp changes

**Pull request merged**